app/views/bookings/index.html.erb

```diff
@@ -4,5 +4,6 @@
   <p><%= booking.id %></p>
   <p><%= animal.name %></p>
```

Comment:
you don't actually need to have line 2 and 3 to find the animal and user. you could simply say booking.animal and booking.user to get those instances. line 5 could thus be 'booking.animal.name' instead of 'animal.name' plus line 2

Reply:
Deleted the extra lines and changed this accordingly.